src/components/EntityScorecardsCard.tsx

```diff
@@ -0,0 +1,231 @@
+import React, { useState } from "react";
+import { InfoCard } from "@backstage/core-components";
+import { useEntity } from "@backstage/plugin-catalog-react";
+import useAsync from "react-use/lib/useAsync";
+import { useApi } from "@backstage/core-plugin-api";
+import { Progress, ResponseErrorPanel } from "@backstage/core-components";
+import Box from "@material-ui/core/Box";
+import Tabs from "@material-ui/core/Tabs";
+import Tab from "@material-ui/core/Tab";
+
+import { dxApiRef, Scorecard, ScorecardCheck } from "../api";
+import { BrandedCardTitle } from "./BrandedCardTitle";
+import { CheckResultBadge } from "./CheckResultBadge";
+import { COLORS } from "../styles";
+
+const DEFAULT_NO_LEVEL_COLOR = "#CBD5E1";
+
+type EntityScorecardsCardProps = {
+  contentMaxHeight?: string | number;
+};
+
+export function EntityScorecardsCard({
+  contentMaxHeight = "20rem",
+}: EntityScorecardsCardProps) {
+  const dxApi = useApi(dxApiRef);
+
+  const { entity } = useEntity();
+
+  const entityIdentifier = entity.metadata.name;
```

Comment:
This makes a big assumption that the DX entity's identifier will match the Backstage entity's metadata name. Are we happy with that for now? Or is there a more flexible way we can turn that mapping over to the plugin user?

Reply:
My imagined solution when we built our new service catalog was aliases.

So I think we could check for an alias and then exact match and then we 404. 

I think we can do this in our API on the DX side, we just need to pass the string entity reference.

Reply:
See [this Slack message](https://getdx.slack.com/archives/C075HBGLDQS/p1743435681976409?thread_ts=1743199654.063199&cid=C075HBGLDQS) - we decided to go with this approach for now